### PR TITLE
{vis}[intel/2015a] libXScrnSaver 1.2.2 + libXft 2.3.2

### PR DESCRIPTION
**Pull request by boegel**
deps for another upcoming PR

### CODE REVIEW

File: easybuild/easyconfigs/l/libXrender/libXrender-0.9.9-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [XORG_LIB_SOURCE]
+
+builddependencies = [
+    ('renderproto', '0.11'),
+]
+
+dependencies = [
+    ('kbproto', '1.0.7'),
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/l/libXrender/libXrender-0.9.9-intel-2015a.eb]: isn't this just a build dependency?

File: easybuild/easyconfigs/l/libXrender/libXrender-0.9.9-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    ('libX11', '1.6.3', '-Python-2.7.9'),
+]
+
+sanity_check_paths = {
+    'files': ['lib/libXrender.a'],
+    'dirs': [],
+}
+
+moduleclass = 'vis'
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/l/libXrender/libXrender-0.9.9-intel-2015a.eb]: these are only libs, so devel instead of vis?

**Comment by JensTimmerman** [comment]
ok to go after remarks are addressed.

File: easybuild/easyconfigs/l/libXrender/libXrender-0.9.9-intel-2015a.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/l/libXrender/libXrender-0.9.9-intel-2015a.eb]: it's a library for visualisation, I feel it's a good idea to group the X11 stuff under 'vis'

Comment (JensTimmerman) [inline easybuild/easyconfigs/l/libXrender/libXrender-0.9.9-intel-2015a.eb]: I was under the impression the module classes were put in place to help users find what they needed. 
So vis would contain programs to visualize things, and devel would contain libraries and headers. But I didn't make up these rules, so feel free to leave this if you please.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 5 out of 6 (6 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1b90f06b1c224ea82876 for a full test report.

File: easybuild/easyconfigs/l/libXrender/libXrender-0.9.9-intel-2015a.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/l/libXrender/libXrender-0.9.9-intel-2015a.eb]: What we really need is support to list multiple moduleclasses, so we can fight over which ones should be listed first. ;)

File: easybuild/easyconfigs/l/libXrender/libXrender-0.9.9-intel-2015a.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/l/libXrender/libXrender-0.9.9-intel-2015a.eb]: it is, thx

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9f86d78ac55b3052d65d for a full test report.

**Comment by boegel** [comment]
Thanks for the review @JensTimmerman!

**Pull request merged**